packages/pigeon/lib/dart_generator.dart

```diff
@@ -248,21 +254,26 @@ void generateDart(DartOptions opt, Root root, StringSink sink) {
             final Field field = klass.fields[index];
             indent.write('..${field.name} = ');
             if (customClassNames.contains(field.dataType)) {
-              indent.add(
-                'pigeonMap[\'${field.name}\'] != null ? ${field.dataType}.decode(pigeonMap[\'${field.name}\']$unwrapOperator) : null',
-              );
+              indent.addln('pigeonMap[\'${field.name}\'] != null');
+              indent.inc(2);
+              indent.writeln(
+                  '? ${field.dataType}.decode(pigeonMap[\'${field.name}\']$unwrapOperator)');
+              indent.write(': null');
```

Comment:
Can you put this back the way it was please.  I know this generates code that is more properly formatted but the pigeon code is harder to maintain.  I prefer to have the pigeon code easier to maintain and just ask people to run a formatter on the generated code (as long as it isn't crazy).  Maybe indent.format would be usable here instead?

Reply:
I mainly did this because otherwise the GitHub Actions kept failing. Should we change the Actions instead to allow for badly formatted code?

Reply:
Na, you can run a formatter on the generated code that jives with your CI, or you can turn off CI for generated code.  It's impossible to make everyone's CI settings happy.

I tweaked the Indent.format so that the output is the same without inc and dec, so in this instance it will be like you expected.